galeracluster/source/glb.rst

```diff
@@ -0,0 +1,154 @@
+.. _`glb-doc`:
+
+Galera Load Balancer provides simple TCP connection balancing developed with scalability and performance in mind, by Codership, Oy.  It draws on Pen for inspiration, but its functionality is limited to only balancing TCP connections.  Galera Load Balancer provides:
+
+- Support for configuring back-end servers at runtime.
+- Support for draning servers. 
+- Support for the epol API for routing performance.
+- Support for multithreaded operations.
+- Optional watchdog module to monitor destinations and adjust the routing table.
+
+------------------------
+Installation
+------------------------
+.. _`glb-install`:
+
+Unlike Galera Cluster, there is no binary installation available for Galera Load Balancer.  Installing it on your system requires that you build it from source.  It is available on GitHub at `glb <https://github.com/codership/glb>`_.
+
+To build Galera Load Balancer, complete the following steps:
+
+#. From a directory convenient to you for source builds, such as ``/opt``, use Git to clone the GitHub repo for Galera Load Balancer.
+
+   .. code-block:: console
+
+      $ git clone https://github.com/codership/glb
+   
+#. Change into the new ``glb/`` directory created by Git, then run the bootstrap script.
+
+   .. code-block:: console
+
+      $ cd glb/
+      $ ./bootstrap.sh
+
+#. Configure Make to build on your system.
+
+   .. code-block:: console
+
+      $ ./configure
+
+#. Build the application with Make.
+
+   .. code-block:: console
+
+      $ make
+
+#. Install the application on your system.
+
+   .. code-block:: console
+
+      # make install
+   
+   .. note:: Galera Load Balancer installs in ``/usr/sbin``.  You need to run the above command as root.
+
+Galera Load Balancer is now installed on your system.  You can launch it from the command-line, using the ``glbd`` command.
+
+In addition to the system daemon, you have also installed ``libglb``, a shared library for connection balancing with any Linux applications that use the ``connect()`` call from the C Standard Library.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Service Installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _`glb-service`:
+
+The above installation procedure only installs Galera Load Balancer to be run manually from the command-line.  However, you may find it more useful to run this application as a system service.
+
+In the source directory you cloned from GitHub, navigate into the ``files/`` directory.  Within this directory there is a configuration file and a service script that you need to copy to their relevant locations.
+
+- Place ``glbd.sh`` into ``/etc/init.d`` directory under a service name.
+
+  .. code-block:: console
+
+     # cp glbd.sh /etc/init.d/glb
+
+- Place ``glbd.cnf`` into either configuration directory.  For Red Hat and its derivatives, this is ``/etc/sysconfig/glbd/``.  For Debian and its derivatives, use ``/etc/default/glbd/``.
+
+  .. code-block:: console
+
+     # cp glbd.cnf /etc/sysconfig/glbd/
```

Comment:
same here. All mentions of the config file should be about glbd.cfg as this is the name of the file in the source repository.